### PR TITLE
Update create-secure-guest-sharing-environment.md

### PR DESCRIPTION
**Pull request by msbemba**
Fixed Typo from ony to only

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3679

File: microsoft-365/solutions/create-secure-guest-sharing-environment.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ You can reduce your attack surface and ease administration by requiring guest us
 
 For Microsoft 365 Groups and Teams, this is done with an Azure AD conditional access policy. For SharePoint, this is configured in the SharePoint admin center. (You can also [use sensitivity labels to restrict guests to web-only access](https://docs.microsoft.com/microsoft-365/compliance/sensitivity-labels-teams-groups-sites).)
 
-To restrict guests to web-ony access for groups and teams
+To restrict guests to web-only access for groups and teams
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/create-secure-guest-sharing-environment.md]:
```suggestion
To restrict guests to web-only access for Groups and Teams:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Pull request merged**